### PR TITLE
Removed dead code

### PR DESCRIPTION
**Pull request by rajesh-vignet**

### CODE REVIEW

File: ContactTest/ContactTest/ViewController.m [hunk 1]
```diff
@@ -122,26 +122,8 @@ - (IBAction)didSelectedAddContact:(id)sender {
 
 - (IBAction)didSelectedEditContact:(id)sender
 {
-//    var contactController = CNContactViewController.FromContact(contact);
-//				contactController.AllowsActions = false;
```
Comment (kvishnuvc) [inline ContactTest/ContactTest/ViewController.m]: ok

**Review by kvishnuvc — CHANGES_REQUESTED** [review]
Put some comment over there

File: ContactTest/ContactTest/ViewController.m (hunk 1 above)
Comment (rajesh-vignet) [reply ContactTest/ContactTest/ViewController.m]: Thanks

**Comment by rajesh-vignet** [comment]
If you are okay with the changes, we can ,merge this.

**Review by kvishnuvc — APPROVED** [review]

**Pull request merged**